code/modules/tgui_panel/tgui_panel.dm

```diff
@@ -86,9 +89,18 @@
 			),
 		))
 		return TRUE
+
 	if(type == "audio/setAdminMusicVolume")
 		client.admin_music_volume = payload["volume"]
 		return TRUE
+
+	if(type == "audio/protected")
+		if(!GLOB.admins_warned)
+			message_admins(span_notice("Audio returned a protected playback error."))
```

Comment:
Also this alert doesn't tell the admins much. Like what are they supposed to do about it